### DISCUSSION TITLE
Trying to compile nasm, home/runner/.local/bin does not exist when running linker

### DISCUSSION THREAD

**Discussion by TheLazyDutchman** [actions/virtual-environments#5290] [General questions] [answered]
Hi, I am trying to use actions to compile generated assembly files from my own language. But when my language tries to link it it says `Error:  could not open dir 'home/runner/.local/bin', error code: 2`. When searching on internet it seemed like I could just install the gnu linker by running `sudo apt-get install gcc-arm-none-eabi`. I also tried to run `echo "home/runner/.local/bin" >> $GITHUB_PATH` but this does not seem to work.

maybe I need to initialize linker after I run `- uses: actions/checkout@v2`, but I somewhat doubt it. This is my first time using actions, so I  probably forgot something stupid, but I have no idea what it might be.

**Comment by miketimofeev** [comment, marked as answer]
Hi @TheLazyDutchmen!
Actions related questions can be asked in the community support forum https://github.community/
